.github/workflows/5_builderpackage_server.yml

```diff
@@ -0,0 +1,24 @@
+run-name: Build wazuh-server packages on PR ${{ github.event.pull_request.number }}
+name: Build wazuh-server packages on PR
+
+on:
+  pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
+    branches:
+      - main
+      - master
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
The workflow `packages-build-server.yml` does not exist in 4.x branches and the build will fail, they are using `packages-build-manager.yml`. Perhaps we should narrow the scope to 5.0 integration tests only?